packages/sdk/src/EventSenderEngine.test.ts

```diff
@@ -63,7 +63,36 @@ describe('EventSenderEngine unit tests', () => {
           eventDefinition: 'my_event',
           eventTime: expect.any(String),
           payload: {
-            a: 0,
+            context: { value: 2 },
+            message: 1,
+          },
+        },
+      ],
+    });
+  });
+  it('payload should prioritize message context field', async () => {
```

Comment:
Is this really what we want? Shouldn't we change the message type to have a reserved key for "context"?